### DISCUSSION TITLE
Is introspection the recommended way to "share" the prisma client?

### DISCUSSION THREAD

**Discussion by dyuan-boxlabs** [prisma/prisma#13765] [Q&A]
I have multiple services that connect to the db and they all (should) have the most up to date prisma client. Ideally I would like one code repo that's dedicated to db migrations and the only way I see to share the schema is to have the other services do prisma db pull.

I just want confirmation if this is the recommended way to share the client or if there is another method.

**Comment by nurul3101** [comment]
Hey @dyuan-boxlabs 👋 
You could create an npm package which would export your latest types and PrismaClient based on your models, you could then import these into all of your apps. This is one of the most straightforward approach I have seen people use.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
